c/floats-cdfpl/newton_1_3_true-unreach-call.i

```diff
@@ -12,7 +12,7 @@ float fp(float x)
 
 int main()
 {
-  float IN;
+  float IN = __VERIFIER_nondet_float();
```

Comment:
At least in this file there is no declaration of `__VERIFIER_nondet_float`.

I did not check other files.